data/io.github.TransmissionRemoteGtk.appdata.xml.in.in

```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2014 Alan F -->
 <component type="desktop">
-  <id>io.github.TransmissionRemoteGtk</id>
+  <id>@app_id@</id>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0+</project_license>
-  <translation type="gettext">transmission-remote-gtk</translation>
-  <name>Transmission Remote</name>
-  <developer_name>Transmission Remote Gtk Team</developer_name>
+  <translation type="gettext">@project_name@</translation>
+  <name>@fancy_name@</name>
```

Comment:
This is just bad for translations. I really don't like this change and I think it should be reverted as it was.

Reply:
I did not know that, easy enough to drop the commit